setup.py

```diff
@@ -17,7 +17,7 @@
 from setuptools import setup, find_packages, Extension
 from setuptools.command.build_ext import build_ext
 
-if sys.version_info[:2] < (2, 7) or (sys.version_info[:1] == 3 and sys.version_info[:2] < (3, 5)):
+if sys.version_info[:2] < (2, 7) or (sys.version_info[:1][0] == 3 and sys.version_info[:2] < (3, 5)):
```

Comment:
Isn't this simpler and easier to read?

```suggestion
if sys.version_info[:2] < (2, 7) or ((3, 0) <= sys.version_info[:2] < (3, 5)):
```